dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/SignIn/TestBase.CommonTests.cs

```diff
@@ -149,4 +149,20 @@ public async Task JourneyIsAlreadyCompleted_DoesNotRedirectToPostSignInUrl(HttpM
             Assert.Null(doc.GetElementByTestId("JourneyExpiredError"));
         }
     }
+
+    public async Task ValidRequest_RendersContent(string url)
+    {
+        // Arrange
+        var authStateHelper = await CreateAuthenticationStateHelper(c => c.Start());
+
+        var fullUrl = new Url(url).SetQueryParam(AuthenticationStateMiddleware.IdQueryParameterName, authStateHelper.AuthenticationState.JourneyId);
+        var request = new HttpRequestMessage(HttpMethod.Get, fullUrl);
+
+        // Act
+        var response = await HttpClient.SendAsync(request);
+
+        // Assert
+        response.EnsureSuccessStatusCode();
+        await response.GetDocument();
```

Comment:
asserts smart quotes used. Helper method only for pages in SignIn for now, and only for those methods that don't try and assert anything else about the content of the page